sympy/vector/deloperator.py

```diff
@@ -49,23 +35,15 @@ def gradient(self, scalar_field, doit=False):
 
         >>> from sympy.vector import CoordSysCartesian
         >>> C = CoordSysCartesian('C')
-        >>> C.delop.gradient(9)
```

Comment:
This one should now return zero. Maybe we can leave it as an example (just change the output).